app/src/main/res/values-nl/strings.xml

```diff
@@ -87,21 +87,21 @@
     <string name="set_as_wallpaper">Als achtergrond instellen</string>
     <string name="set_as_wallpaper_failed">Achtergrond instellen mislukt</string>
     <string name="set_as_wallpaper_with">Achtergrond instellen met:</string>
-    <string name="no_capable_app_found">Geen app gevonden voor instellen achtergrond</string>
+    <string name="no_capable_app_found">Geen app gevonden voor instellen van achtergrond</string>
     <string name="setting_wallpaper">Achtergrond instellen&#8230;</string>
     <string name="wallpaper_set_successfully">Achtergrond is ingesteld</string>
     <string name="portrait_aspect_ratio">Verhouding in portretmodus</string>
     <string name="landscape_aspect_ratio">Verhouding in landschapsmodus</string>
-    <string name="home_screen">Startscherm</string>
-    <string name="lock_screen">Vergrendelingsscherm</string>
-    <string name="home_and_lock_screen">Start- en vergrendelingsscherm</string>
+    <string name="home_screen">Beginscherm</string>
+    <string name="lock_screen">Vergrendelscherm</string>
```

Comment:
I thought vergrendelingsscherm was the official Android translation, I could be wrong.

Reply:
It says "vergrendelscherm" on my Android tablet, but maybe there are differences b/w Android versions.

Reply:
Ah ok, that's it then. Ok "vergrendelscherm" is more concise, cool.